packages/pn-personafisica-webapp/src/components/Contacts/DigitalContactElem.tsx

```diff
@@ -238,10 +251,16 @@ const DigitalContactElem = forwardRef<{ editContact: () => void }, Props>(
                   onClick={toggleEdit}
                   sx={{ mr: 2 }}
                   disabled={editDisabled}
+                  id="modifyContact"
                 >
                   {t('button.modifica')}
                 </ButtonNaked>
-                <ButtonNaked color="primary" onClick={removeHandler} disabled={editDisabled}>
+                <ButtonNaked
+                  id="cancelContact"
```

Comment:
Be aware this id is not unique. You can use the concatenation between this string and f.id

Reply:
same as before

Reply:
ok use the concatenation with senderId